docker/docker-prepare.sh

```diff
@@ -54,7 +78,9 @@ superuser() {
 }
 
 do_work() {
-	if [[ -n "${PAPERLESS_DBHOST}" ]]; then
+  if [[ "${PAPERLESS_DBENGINE}" == "mariadb" ]]; then
```

Comment:
So, if PAPERLESS_DBENGINE is empty, it defaults to postgresql?

Should we make this explicit?
This will add a regression for all users, using postgres though..

Reply:
Correct, all pre-existing behavior currently remains; if you don't specify the engine, it will default to postgresql. I would be happy to make it explicit, but I opted in favor of backwards compatibility for running with postgresql. I just didn't want to rock the boat or get in anyone's way with this change.